core/color/solids.json

```diff
@@ -1,64 +1,124 @@
 {
   "color": {
+    "bronze": {
```

Comment:
In the momentum-design/tokens implementation, solid colours exist under: color: {solid: {*.....}}
This one is adding them directly under color
color: {*....}

Reply:
the automation is adding all solid colors twice :(

Reply:
Ah that one is an easy fix. I can do that